dask/dataframe/partitionquantiles.py

```diff
@@ -429,6 +426,8 @@ def percentiles_summary(df, num_old, num_new, upsample, state):
     except (TypeError, NotImplementedError):
         vals, _ = _percentile(array_safe(data, data.dtype), qs, interpolation)
 
+    vals = list(np.nan if v is pd.NA else v for v in vals)
```

Comment:
Ideally, this conversion could be done conditionally based on the value of `vals._hasna` (assume `vals` is an `ExtensionArray`), but for older versions of pandas we're dealing with a numpy array here instead, for which I wasn't able to find any trivial way to check for containment of `pd.NA`